src/config/env/prd.json

```diff
@@ -28,6 +28,7 @@
   "SUBMISSION_THRESHOLD_PITCH": "100",
   "SUBMISSION_THRESHOLD_TENDER": "2500",
   "SUBMISSION_THRESHOLD_GRANT": "100",
+  "SUBMISSION_THRESHOLD_COUNCIL_DECISION_VETO": "25000",
```

Comment:
is 25k vp threshold for submission ok?

Reply:
Yess, check https://github.com/decentraland/governance/issues/1900